src/types.ts

```diff
@@ -1,44 +1,39 @@
-export type VoiceStartMessage = {
-  type: 'voiceStart';
+export type MeasureStartMessage = {
+  msgType: 'measureStart';
+  width: string;
+  staves: string;
+  clef: string[];
```

Comment:
nit: Since there are multiple, can we call this `clefs` instead?

Reply:
done